shelley-ma/formal-spec/shelley-ma.tex

```diff
@@ -102,6 +102,8 @@
 \newcommand{\seedOp}{\star}
 \newcommand{\Ppm}{\type{Ppm}}
 \newcommand{\Value}{\type{Value}}
+\newcommand{\Integer}{\type{Integer}}
+\newcommand{\ValClass}{\mathcal{VAL}}
```

Comment:
I don't like how this is typeset, multiple capital mathcal letters together often look weird. Maybe `\mathbf{Val}`?